redbeams-api/src/main/java/com/sequenceiq/redbeams/api/model/common/DetailedStackStatus.java

```diff
@@ -0,0 +1,21 @@
+package com.sequenceiq.redbeams.api.model.common;
+
+public enum DetailedStackStatus {
```

Comment:
Though this is a cf stack and the name is not bad, but just to avoid name collision with the status of stack/cluster I would give a different name. Like DetailedDatabaseServerStatus.